### DISCUSSION TITLE
Documentation page support for searching all icons

### DISCUSSION THREAD

**Discussion by StringKe** [ng-icons/ng-icons#81] [Ideas]
I try to find an icon and I need to look for it many times in different icon libraries, support for searching all icons would be more convenient.

**Comment by ashley-hunter** [comment]
Hi, thanks for the suggestion! I originally didn't add this because it was quite slow - searching through 30,000+ icons and potentially rendering thousands of matches, but I have since switched to use `rxFor` for more performant rendering, so this may not be a problem any more. I will look into it when I get some time, thank you.

**Comment by carlmontplaisir-beslogic** [comment]
This would be very useful as a feature on the website. If performance is an issue, manually selecting multiple icon library would be great. That way we could select the libraries we've imported in our projects. Thanks for this great library!

**Reply by ashley-hunter** [reply]
Thank you! Yes, its always going to be challenge with potentially 50,000+ icons to be rendered. I will take a look into it and see what the performance is like and if there are any other challenges.
